File: C2_-_Input_processing_and_output/E2_-_Character_count/CPP/main.cpp

#include <iostream>
#include <string>

int main ( void )
{
    std::string line;

    std::cout << "Enter a string: " << line.capacity();

    std::getline( std::cin, line );

    std::cout << "There are " << line.length() << " in the string." << std::endl;

    return 0;
}

File: C2_-_Input_processing_and_output/E2_-_Character_count/RUST/main.rs
use std::io::stdin;
use std::io::Write;
use std::io;

fn main()
{
    println!( "Enter a string -->" );
    io::stdout().flush().unwrap();
    let mut line = String::new();

    let _ret = stdin().read_line( &mut line );

    println!( "There are {} character in the string", line.trim_end().len() );
}